test/examples/LimitOrderProtocol-example.js

```diff
@@ -0,0 +1,284 @@
+const hre = require('hardhat');
+const { ethers } = hre;
+const { expect, time, constants, trim0x } = require('@1inch/solidity-utils');
+const { fillWithMakingAmount, buildMakerTraits, buildOrder, signOrder, compactSignature, ABIOrder } = require('../helpers/orderUtils');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+const { joinStaticCalls, ether, cutLastArg } = require('../helpers/utils');
+
+describe.skip('LimitOrderProtocol usage example', function () {
```

Comment:
Is this by design (`.skip`)?

Reply:
Yes, this is an example of usage. So these tests  are not intended to run on each pass.